README.md

```diff
@@ -91,6 +91,7 @@ The easiest way to do this is using the docker instructions below.
 Populate your database:
 
     $ ./manage.py sync_couch_views
+	$ ./manage.py create_kafka_topics
```

Comment:
nitpick: there's a <tab> character here whereas our convention is a 4-space "soft tab" (i.e. 4 spaces instead of a <tab> character). You can (and should) configure the editor you're using to use 4 spaces instead of a tab character when you hit <tab>.